python/ray/serve/controller.py

```diff
@@ -428,6 +430,12 @@ def _create_control_loop_metrics(self):
             {"actor_id": ray.get_runtime_context().get_actor_id()}
         )
 
+    def _count_start(self):
```

Comment:
nit: this method is only wrapping a single method call so it's probably unnecessary indirection

Reply:
Makes sense, I removed the indirection.